### PR TITLE
Minor amendments to docs

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
Add OSX steps.

### CODE REVIEW

**Comment by jmoldon** [comment]
I think it is easier for Mac users if they have all the commands they need together (without having to understand what "as above" means).

So I would duplicate the information in this case.

[Can I modify this pull request with that change?]

File: docs/quickstart.md [hunk 1]
```diff
@@ -21,19 +21,19 @@ If you don't have a conda environment on your system, this will create a tempora
...
 cd droplets
-curl https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O
+curl -O https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
 bash ./Miniconda3-latest-Linux-x86_64.sh -b -p my_conda_env
 rm ./Miniconda3-latest-Linux-x86_64.sh
 source my_conda_env/etc/profile.d/conda.sh 
 ```
 ```bash
-conda create -n droplets python=3.6
+conda create env -f environment.yml
```
Comment (jmoldon) [inline docs/quickstart.md]: conda env create -f environment.yml

**Review by jmoldon — CHANGES_REQUESTED** [review]
There is a typo in line 30

**Comment by sebastian-luna-valero** [comment]
I have updated the docs as requested. Please feel free to add your suggestions.

**Pull request merged**